Tests/SwiftParserTest/Expressions.swift

```diff
@@ -147,6 +155,7 @@ final class ExpressionTests: XCTestCase {
       )
       """,
       diagnostics: [
+        DiagnosticSpec(message: "Expected expression after ':' in dictionary"),
         DiagnosticSpec(message: "Expected ']' to end dictionary"),
```

Comment:
Same as above. FWIW the current diagnostic is a missing dictionary value (`expected value in dictionary literal`), but I think it's fine not to special case it.

There's obviously quite a few of these cases so I won't mention any others 😅